daemon/openrazer_daemon/hardware/mouse_mat.py

```diff
@@ -24,3 +24,41 @@ class RazerFirefly(__RazerDeviceBrightnessSuspend):
         "side_img": "https://assets.razerzone.com/eeimages/products/21936/fireflycloth-gallery-6.png",
         "perspective_img": "https://assets.razerzone.com/eeimages/products/21936/fireflycloth-gallery-3.png"
     }
+
+
+class RazerGoliathus(__RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Goliathus (2018)
+    """
+    USB_VID = 0x1532
+    USB_PID = 0x0C01
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 1]
```

Comment:
`[1, 1]`? Sounds a bit wrong

Reply:
My understanding is that this is for custom matrix commands. Goliathus allows such commands, but you only get 1 addressable LED...